src/libraries/System.Text.Json/codegenerator.Tests/JsonSerializerSourceGeneratorTests.cs

```diff
@@ -0,0 +1,15 @@
+﻿using Xunit;
+
+namespace System.Text.Json.CodeGenerator.Tests
+{
+    // TODO(@kevinwkt): Temporary end2end tests to use the generated code using codegen.
```

Comment:
nit: we don't typically add GitHub usernames to TODOs.

Where necessary, we add links to GitHub issues. You can open issues on this repo based on the task items on the Json code gen epic which we'll share with the community.